actions/image_partition_action.go

```diff
@@ -54,7 +54,8 @@ Mandatory properties:
 
 - name -- is used for referencing named partition for mount points
 configuration (below) and label the filesystem located on this partition. Must be
-unique.
+unique. The partition name can be up to 11 characters long for vfat and 16 characters
+long for ext2/3/4. Name should be same as partition name in mount points.
```

Comment:
```suggestion
long for ext2/3/4. Any longer labels will be automatically truncated.
```

also; the length should probably be moved to the `partlabel` documentation